proto/kvrpcpb.proto

```diff
@@ -993,3 +993,22 @@ message GetLockWaitInfoResponse {
     string error = 2;
     repeated deadlock.WaitForEntry entries = 3;
 }
+
+message RawCoprocessorRequest {
+    kvrpcpb.Context context = 1;
+
+    string copr_name = 2;
+    // Coprorcessor version constraint following SEMVER definition.
+    string copr_version_req = 3;
+
+    repeated KeyRange ranges = 4;
+    bytes data = 5;
+}
+
+message RawCoprocessorResponse {
+    errorpb.Error region_error = 1;
+    // Error message for cases like if no coprocessor with a matching name is found
+    // or on a version mismatch between plugin_api and the coprocessor.
+    string error = 2;
```

Comment:
Ah nice, I noticed as well that `other_error` doesn't play well with the macros :)